hkm/static/hkm/scss/_app.scss

```diff
@@ -421,7 +421,7 @@ span.no-postal-fee {
   width: 100%;
   height: 60%;
   flex-direction: row;
-  min-height: 700px;
+  min-height: 800px;
```

Comment:
As word `image` was changed to `picture` the sentence was overflowing outside its container. Adjusting `min-height` fixes this.